project/Dependencies.scala

```diff
@@ -19,4 +19,7 @@ object Dependencies {
     "com.ibm.wala" % "com.ibm.wala.util" % "1.4.2",
     "com.ibm.wala" % "com.ibm.wala.core" % "1.4.2"
   )
+
+  lazy val sonatypeResolver = "Sonatype OSS Releases" at "https://oss.sonatype.org/content/repositories/releases"
+  lazy val scalaSMTLIB = "com.regblanc" %% "scala-smtlib" % "0.2.1"
```

Comment:
Dependency questions:
  - Are we requiring z3 as an external dependency for the whole project?
  - Is z3 installed automatically by scala-smtlib or the user must install it manually?
  - What does it happen if one does not have z3 installed on his machine?

We should understand the better way of adding it as dependency.